app/build-js.sh

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
+echo "Setting up path variables and finding node"
 MYPATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-ROOT=$MYPATH/..
-EXTRA_NPM_ARGS="--no-progress --no-spin"
+EXTRA_NPM_ARGS=""
```

Comment:
I copied the `build-js.sh` file from `astro-scaffold`. Actually I don't think `EXTRA_NPM_ARGS` is even used anymore since we don't call `npm install` in the script. Not sure why it was left in scaffold?! @jvoll @jeremywiebe was there a reason we still have this variable in scaffold?